src/schemata/series.schema.json

```diff
@@ -0,0 +1,142 @@
+      "items": {
+        "type": "string"
+      }
+    },
+    "games": {
+      "description": "Games belonging to this series",
+      "type": "object",
+      "patternProperties": {
+        ".": {
+          "type": "object",
+          "description": "Game",
+          "properties": {
+            "genres": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "enum": ["Adventure", "RPG", "Platformer", "Shooter", "Strategy", "Puzzle", "Survival"]
+              }
+            },
+            "platforms": {
+              "type": "array",
+              "items": {
+                  "type": "string",
+                  "enum": [
+                    "PC",
+                    "NES", "SNES", "N64", "GameCube", "Wii", "Wii U", "Switch",
+                    "GameBoy", "GBA", "DS", "3DS",
+                    "Master System", "Genesis", "Saturn", "Dreamcast",
+                    "PS1", "PS2", "PS3", "PS4", "PS5",
+                    "PSP", "Vita",
+                    "Xbox", "Xbox 360", "Xbox One", "Xbox Series",
+                    "Arcade", "MSX"
+                  ]
+              }
+            },
+            "release-date": {
+              "type": "string",
+              "format": "date"
+            },
+            "sub-series": {
+              "type": "string",
+              "description": "Key described in Series"
+            },
+            "comment": {
+              "type": "string",
+              "description": "An optional game description formatted with Markdown"
+            }
+          },
+          "required": [
+            "genres",
+            "platforms"
+          ],
+          "additionalProperties": false
+        }
+      }
+    },
+    "randomizers": {
+      "type": "array",
+      "description": "Randomizers that can randomize games in this series",
+      "items": {
+        "type": "object",
+        "properties": {
+          "games": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "identifier": {
+            "description": "Name of the randomizer - should be distinguishable from others within the series.",
+            "type": "string"
+          },
+          "url": {
+            "description": "Link to randomizer download or info page",
+            "type": "string",
+            "format": "uri"
+          },
+          "multiworld": {
+            "description": "Identifier of the multi-series randomizer this belongs to",
+            "type": "string"
+          },
+          "updated-date": {
+            "description": "Last time this entry was updated",
+            "type": "string",
+            "format": "date",
+            "default": "1900-01-01"
+          },
+          "added-date": {
+            "description": "Date this randomizer was added",
+            "type": "string",
+            "format": "date",
+            "default": "1900-01-01"
```

Comment:
I don't think we need this default since we filled them all in, and new entries should definitely have it. I removed this line and it still succeeded for me.


```suggestion
```

Reply:
`default` in JSON-schema is not a 'null values get coerced to this', but rather a 'missing values are semantically equivalent to this'. Afaik no validators utilise `default` at all. In fact, the spec [only recommends it even be a valid value](https://json-schema.org/draft/2020-12/draft-bhutton-json-schema-validation-00#rfc.section.9.2)